content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -136,8 +136,8 @@ If the container runtime of choice is Docker, it is used through the built-in
 
 Other CRI-based runtimes include:
 
-- [containerd](https://github.com/containerd/cri) (CRI plugin built into containerd)
-- [cri-o](https://cri-o.io/)
```

Comment:
I'd leave this as is, I think it works better.

Reply:
Alright, I reverted that change. :+1: